### ISSUE TITLE
fine-tune model for commercial use?

### ISSUE THREAD

**Issue by lw3259111** [EleutherAI/gpt-neox#861]
We would like to fine-tune your model, and we are wondering if the fine-tuned model can be used for commercial purposes.

**Comment by satpalsr** [comment]
Yes you can. [Read the License for more info](https://github.com/EleutherAI/gpt-neox/blob/main/LICENSE).

**Comment by Quentin-Anthony** [comment]
> Yes you can. [Read the License for more info](https://github.com/EleutherAI/gpt-neox/blob/main/LICENSE).

Yep all of our code and models are permissively licensed for research and commercial use.
